### PR TITLE
fix(dedicated): fix tracking on 3az node

### PR DESCRIPTION
**Pull request by qpavy**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/multi-az-feature`
| Bug fix?         | yes
| New feature?     |no
| Breaking change? | no
| Tickets          | Fix #MANAGER-12500
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- ~~[ ] Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~~[ ] Breaking change is mentioned in relevant commits~~

## Description

fix missing tracking for 3az cluster node .

### CODE REVIEW

File: packages/manager/modules/bm-server-components/src/advanced-features/sgx/introduction/introduction.html [hunk 1]
```diff
@@ -35,7 +35,7 @@
                     rel="noopener"
                     data-track-on="click"
                     data-track-type="action"
-                    data-track-name="dedicated::dedicated::server::sgx::go-to-documentation"
+                    data-track-name="{{:: $ctrl.trackingdocumentation }}"
```
Comment (anooparveti) [inline packages/manager/modules/bm-server-components/src/advanced-features/sgx/introduction/introduction.html:38]:
```suggestion
                    data-track-name="{{:: $ctrl.trackingDocumentation }}"
```

File: packages/manager/modules/bm-server-components/src/advanced-features/sgx/manage/confirmation/confirmation.controller.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export default class SgxConfirmationController {
+  $onInit() {
+    this.trackPage(`${this.sgxTrackingPrefix}::manage::confirm`);
+  }
```
Comment (darsene) [inline packages/manager/modules/bm-server-components/src/advanced-features/sgx/manage/confirmation/confirmation.controller.js:4]: Why do you replace 
```
atInternet: {
        rename: 'dedicated::dedicated::server::sgx',
      },
```
with controller.$onInit ? What's the difference ?

**Review by darsene — COMMENTED** [review]
Basically, it works but instead of using a boolean `isServerNode` which can manage only 2 states (so 2 values), you should have used a property `serverType`(`'node'|'server'|...`) to ease the code for all the tracking.
Moreover, if `node` tracking value need to be update, it's really faster.
What's your opinion ?

**Review by anooparveti — APPROVED** [review]

File: packages/manager/modules/bm-server-components/src/service-status/service-status.controller.js [hunk 3]
```diff
@@ -36,7 +36,7 @@ export default class BmServerComponentsDashboardServiceStatusController {
 
   canBeTerminate() {
     return (
-      !this.server.isExpired && !this.server.engagement && !this.serverIsNode
+      !this.server.isExpired && !this.server.engagement && !this.serverType
```
Comment (darsene) [inline packages/manager/modules/bm-server-components/src/service-status/service-status.controller.js:39]:
```suggestion
      !this.server.isExpired && !this.server.engagement && this.serverType !== 'node'
```
rather no ?

**Review by darsene — APPROVED** [review]

**Pull request merged**